gradle.properties

```diff
@@ -1,5 +1,5 @@
 org.gradle.caching=true
-group=org.ballerinalang
+group=io.ballerina
```

Comment:
```suggestion
group=io.ballerina.stdlib
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-jballerina.java.arrays/pull/195/commits/853a715b9e2b754ebddd75a2af46cde9780a6ead